src/library_pthread_stub.js

```diff
@@ -13,7 +13,7 @@ var LibraryPThreadStub = {
 #if MINIMAL_RUNTIME
     return typeof importScripts === 'undefined';
```

Comment:
```suggestion
    return typeof window === 'object';
```

Reply:
Why is this better?   This can go in a separate change I think.  do you feel like uploading one?

Reply:
IIUC, when not doing this, `emscripten_is_main_browser_thread` will still return true on Node when linking with `-sMINIMAL_RUNTIME`. For example, notice how currently `ENVIRONMENT_IS_WEB` and `ENVIRONMENT_IS_WORKER` are auto-detected:
https://github.com/emscripten-core/emscripten/blob/088462430ef8b7f5a98de0fd7ad0f62d9ec47fcb/src/shell.js#L95-L97

(note that this comment wasn't made on the actual change, but on the `MINIMAL_RUNTIME` guard)

Reply:
With the test program from https://github.com/emscripten-core/emscripten/pull/15503#issuecomment-968094217, I see this (on the main branch):
```bash
$ emcc test.c -o test.js
$ node test.js
emscripten_is_main_runtime_thread() = 1
emscripten_is_main_browser_thread() = 0
$ emcc -sMINIMAL_RUNTIME test.c -o test-minimal-runtime.js
$ node test-minimal-runtime.js
emscripten_is_main_runtime_thread() = 1
emscripten_is_main_browser_thread() = 1
```
We could also just remove this minimal runtime guard in favor of `ENVIRONMENT_IS_WEB`, since that should already be defined, see:
https://github.com/emscripten-core/emscripten/blob/9297133fd75b9e91c66eea23bd6c27609eb4724d/src/shell_minimal.js#L45

Reply:
In my mental model `emscripten_is_main_browser_thread()` should return true in Node.js main VM thread, otherwise a lot of the threading subsystem would be broken(?)